tests/proxy_server.py

```diff
@@ -489,11 +489,15 @@ def test(HandlerClass=ProxyRequestHandler, ServerClass=ThreadingHTTPServer, prot
 
 
     HandlerClass.protocol_version = protocol
-    httpd = ServerClass(server_address, HandlerClass)
+    try:
+      httpd = ServerClass(server_address, HandlerClass)
+      sa = httpd.socket.getsockname()
+      print "bind succeded"
```

Comment:
Note typo in the spelling of succeeded, here and throughout this patch.

```suggestion
      print("bind succeeded")
```

Please call [print as a function](https://www.python.org/dev/peps/pep-3105/), not as a statement. I was surprised to see we have uses of print as a statement.

Reply:
I thought I should use print like that because in the `test_proxy_use.py` file is said: 
```
  THIS module requires Python2.7 (not 2.8.x, not 3.x, just 2.7.x) as the test
  proxy it uses only supports Python2.7.
```
and I thought I should use python 2 syntax.
I will fix it.

Reply:
Thanks! Fortunately print as a statement was introduced in Python 2.7, so we don't have to carry code that [Guido has long regretted](https://legacy.python.org/doc/essays/ppt/regrets/PythonRegrets.pdf) 😁